cs-microfocus-dca/src/main/java/io/cloudslang/content/dca/actions/utils/GetSubscriptionParams.java

```diff
@@ -0,0 +1,153 @@
+                    @Output(value = RETURN_CODE, description = RETURN_CODE_DESC),
+                    @Output(value = RETURN_RESULT),
+                    @Output(value = EXCEPTION, description = EXCEPTION_DESC)},
+            responses = {
+                    @Response(text = ResponseNames.SUCCESS, field = RETURN_CODE, value = SUCCESS, matchType = COMPARE_EQUAL),
+                    @Response(text = ResponseNames.FAILURE, field = RETURN_CODE, value = FAILURE, matchType = COMPARE_EQUAL, isDefault = true, isOnFail = true)})
+
+    public Map<String, String> execute(@Param(value = URL, required = true, description = URL_DESC) String url,
+                                       @Param(value = AUTH_TYPE, description = AUTH_DESC) String authType,
+                                       @Param(value = USERNAME, description = USERNAME_DESC) String username,
+                                       @Param(value = PASSWORD, description = PASSWORD_DESC) String password,
+                                       @Param(value = PROXY_HOST, description = PROXY_HOST_DESC) final String proxyHost,
+                                       @Param(value = PROXY_PORT, description = PROXY_PORT_DESC) final String proxyPort,
+                                       @Param(value = PROXY_USERNAME, description = PROXY_USER_DESC) final String proxyUsername,
+                                       @Param(value = PROXY_PASSWORD, encrypted = true, description = PROXY_PASS_DESC) final String proxyPassword,
+                                       @Param(value = TRUST_ALL_ROOTS, description = TRUST_ALL_ROOTS_DESC) final String trustAllRoots,
+                                       @Param(value = X509_HOSTNAME_VERIFIER, description = X509_DESC) final String x509HostnameVerifier,
+                                       @Param(value = TRUST_KEYSTORE, description = TRUST_KEYSTORE_DESC) final String trustKeystoreInp,
+                                       @Param(value = TRUST_PASSWORD, encrypted = true, description = TRUST_PASSWORD_DESC) final String trustPasswordInp,
+                                       @Param(value = KEYSTORE, description = KEYSTORE_DESC) final String keystoreInp,
+                                       @Param(value = KEYSTORE_PASSWORD, encrypted = true, description = KEYSTORE_PASS_DESC) final String keystorePasswordInp,
+                                       @Param(value = CONNECT_TIMEOUT, description = CONNECT_TIMEOUT_DESC) final String connectTimeout,
+                                       @Param(value = SOCKET_TIMEOUT, description = SOCKET_TIMEOUT_DESC) final String socketTimeout,
+                                       @Param(value = USE_COOKIES, description = USE_COOKIES_DESC) final String useCookies,
+                                       @Param(value = KEEP_ALIVE, description = KEEP_ALIVE_DESC) final String keepAlive,
+                                       @Param(value = QUERY_PARAMS, description = QUERY_PARAMS_DESC) String queryParams) {
+
+
```

Comment:
I think should be added default values for trustAllRoots, X509HostnameVerifier, proxyPort